package.json

```diff
@@ -10,13 +10,5 @@
     "build:min": "cd code && yarn gulp vscode-reh-web-linux-x64-min",
     "rebuild-native-modules": "cd code && npm rebuild"
   },
-  "license": "EPL-2.0",
-  "dependencies": {
-    "node-gyp": "^9.4.1"
```

Comment:
Removing `node-gyp` from npm was a breaking change introduced in https://github.com/npm/cli/pull/6932.
They already fixed the problem in https://github.com/npm/cli/pull/6932.

While a new version of Node.js with the fixed npm has not yet been released, the only temporary workaround that we need on the Che-Code side is [installing `node-gyp` globally](https://github.com/che-incubator/che-code/pull/305/files#diff-5a132d90967fc45fd3240e3fe2628e15e5ab93f515e9008945e9ccc08f7ab231R51) to allow any transitive dependency rely on it. Once we update to a newer Node.js, we can get rid of that ^^ workaround.

Anyway, if it was even decided that `node-gyp` was no longer part of npm, we don't need to declare any transitive dependency of VS Code in Che-Code's root package.json.